Gemfile

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+group :test do
+  gem 'rantly', '~> 3.0'
+  gem 'rspec', '~> 3.13'
+end
+
+group :development do
+  gem 'rubocop', require: false
+  gem 'rubocop-rspec', require: false
+end
```

Comment:
I might be missing something, but [according to this rather old article](https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/), even if Gemfile for a library which gets to be built as a gem itself is to be used, it should be just:

```ruby
source 'https://rubygems.org'

gemspec
```

Would that make sense to follow this approach, or you see some contradiction with that?

Reply:
There are different opinions on that, but here I'm kind of following what I did in KSV (see https://github.com/kaitai-io/kaitai_struct_visualizer/commit/f49a18c30b035d8f14a4fbdd71157e8317169f3d), i.e. runtime dependencies to `.gemspec` (but the Ruby runtime doesn't have any at the moment), development dependencies to `Gemfile`. See https://github.com/rubygems/rubygems/issues/1104 for a bunch of rationales both in favor or against this.

Frankly, I don't care as long as it works.

Reply:
Got it, thanks!